os/drivers/video/video_null_driver.c

```diff
@@ -172,24 +169,24 @@ struct video_devops_s dummy_null_video_ops = {
 struct v4l2_format_s supported_formats[] = {
 	{
 		"MJPEG", V4L2_BUF_TYPE_VIDEO_CAPTURE, V4L2_PIX_FMT_MJPEG, V4L2_COLORSPACE_SRGB, 12, {
-			{VIDEO_HSIZE_QVGA, VIDEO_VSIZE_QVGA, V4L2_FRMIVAL_TYPE_DISCRETE, {{333333,}, {500000,}, {666666,}, {1000000,}, {999999,}}, MAX_INTERVAL,},
-			{VIDEO_HSIZE_VGA, VIDEO_VSIZE_VGA, V4L2_FRMIVAL_TYPE_DISCRETE, {{333333,}, {500000,}, {666666,}, {1000000,}, {999999,}}, MAX_INTERVAL,},
-			{VIDEO_HSIZE_QUADVGA, VIDEO_VSIZE_QUADVGA, V4L2_FRMIVAL_TYPE_DISCRETE, {{333333,}, {500000,}, {666666,}, {1000000,}, {999999,}}, MAX_INTERVAL,},
-			{VIDEO_HSIZE_HD, VIDEO_VSIZE_HD, V4L2_FRMIVAL_TYPE_DISCRETE, {{333333,}, {500000,}, {666666,}, {1000000,}, {999999,}}, MAX_INTERVAL,},
-			{VIDEO_HSIZE_FULLHD, VIDEO_VSIZE_FULLHD, V4L2_FRMIVAL_TYPE_DISCRETE, {{333333,}, {500000,}, {666666,}, {1000000,}, {999999,}}, MAX_INTERVAL,},
-			{VIDEO_HSIZE_5M, VIDEO_VSIZE_5M, V4L2_FRMIVAL_TYPE_DISCRETE, {{333333,}, {500000,}, {666666,}, {1000000,}, {999999,}}, MAX_INTERVAL,},
-			{VIDEO_HSIZE_3M, VIDEO_VSIZE_3M, V4L2_FRMIVAL_TYPE_DISCRETE, {{333333,}, {500000,}, {666666,}, {1000000,}, {999999,}}, MAX_INTERVAL,}
+			{VIDEO_HSIZE_QVGA, VIDEO_VSIZE_QVGA, V4L2_FRMIVAL_TYPE_DISCRETE, { {333333,}, {500000,}, {666666,}, {1000000,}, {999999,} }, MAX_INTERVAL,},
```

Comment:
why do you add space?

Reply:
os/drivers/video/video_null_driver.c:173: ERROR: [SPC_M_SEP] space required after that close brace '}'
I just adjusted the space in open  } also for better readability. Should we remove it or not?